main/src/ca/uwaterloo/flix/language/phase/Inliner.scala

```diff
@@ -230,10 +230,28 @@ object Inliner {
       val e1 = visitExp(exp1, subst0)
       val e2 = visitExp(exp2, subst0)
       val e3 = visitExp(exp3, subst0)
```

Comment:
Usually names like e1, e2, and e3 are fine when things are simple. That is not the case here. May I propose:

`outerCond`, `outerThen`, `outerElse`, `innerCond`, `innerThen`, and `innerElse` :)

Reply:
Also, did you consider if this should also happen inside substituteExp or not?

Reply:
(Not saying it has to, just if you considered)

Reply:
I think it's a good idea to have it both places. Seems like 1/4 of the nested ifs are being seen in substituteExp after a quick measurement